doc/change-payment-method-of-payment.md

```diff
@@ -6,10 +6,12 @@ You can change payment method of created payment that is waiting for payment. Th
 
 ```php
 
-/** @var $client \ThePay\ApiClient\TheClient */
+/** @var \ThePay\ApiClient\TheClient $client */
 
-$paymentMethodCode = PaymentMethodCode::TRANSFER;
+// one method selected by user from,
+// list loaded by SDK method: $client->getActivePaymentMethods->all()[0]->getCode()
+$methodCode = 'transfer';
```

Comment:
stejne se obavam ze nekoho napadne si zjistit cody a vytvorit si vlastni enum :D Mozna by pomohlo sem dat odkaz na pripadne ziskani vsech metod codes

Reply:
Good idea remove all constant method codes from examples, it can lead developers into implementation where hardcoded codes are OK.

I make new md file, for [method selection](https://github.com/ThePay/api-client/pull/46/files#diff-126ee5f2a3a8710873de7458fb9e940861f8cfe85aad55ee7d684fc7873c56f7), because i not find any.

Reply:
yea sounds good